symengine/python/symengine/lib/symengine_wrapper.pyx

```diff
@@ -1698,9 +1745,374 @@ def powermod_list(a, b, m):
         s.append(c2py(<RCP[const symengine.Basic]>(v[i])))
     return s
 
+# Is this really intentional? eval_double already defined earlier:
```

Comment:
We'll keep the earlier definition there as the other eval definitions are there and remove the one that is here.